### ISSUE TITLE
Unexpected :exports behavior in org-mode.

### ISSUE THREAD

**Issue by GregorySchwartz** [syl20bnr/spacemacs#8927]
#### Description :octocat:
In org mode for SRC blocks, ":exports none" and ":exports results" both are ignored and behave
like "exports :both"

#### Reproduction guide :beetle:
- Start Emacs
- example.org:

```
# NAME: example
# HEADER: :exports results
# BEGIN_SRC sh
echo "hi"
# END_SRC

# RESULTS: example
: hi
```

- `<SPC> m e e h h`
- Open example.html in browser

*Observed behaviour:* :eyes: :broken_heart:
See both the code and the result.

*Expected behaviour:* :heart: :smile:
See the result only.
Exporting with emacs -q results in this behavior, so it is spacemacs specific.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 25.2.1
- Spacemacs: 0.200.9
- Spacemacs branch: master (rev. 8e1af145)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(python nlinum yaml shell-scripts csv html javascript
        (auto-completion :variables auto-completion-return-key-behavior nil auto-completion-tab-key-behavior 'cycle auto-completion-complete-with-key-sequence nil auto-completion-enable-help-tooltip t auto-completion-enable-sort-by-usage t spacemacs-default-company-backends
                         '(company-capf company-dabbrev-code company-gtags company-etags company-keywords company-files company-dabbrev))
        helm better-defaults emacs-lisp git markdown mu4e
        (org :variables org-enable-reveal-js-support t org-enable-bootstrap-support t)
        (shell :variables shell-default-height 30 shell-default-position 'bottom shell-default-shell 'multi-term shell-default-term-shell "/usr/bin/fish")
        spell-checking syntax-checking version-control erc
        (ranger :variables ranger-show-preview t)
        pdf-tools bibtex
        (haskell :variables haskell-completion-backend 'intero haskell-enable-hindent-style "johan-tibell")
        ess
        (latex :variables latex-enable-auto-fill t latex-enable-folding t)
        pandoc)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GCONF GSETTINGS NOTIFY ACL GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 MODULES

**Comment by kassick** [comment]
Hi,

I don't use spacemacs but I've been seeing this behaviour for some time.

In my setup, it appears that having org-export-babel-evaluate set to nil triggers the issue -- org wants to export code no matter what.

Leaving it with the default value fixed the issue (org-plus-contrib-20170515)

**Comment by GregorySchwartz** [comment]
Found the solution:
https://www.miskatonic.org/2016/10/03/orgexportbabelevaluate/

It looks like org-export-babel-evaluate set to nil ignores headers, even the :exports header. The solution is to have it set to t and instead use

#+PROPERTY: header-args :eval never-export
